wordpress-mobile/WordPress-iOS
# Update fastlane

Fixes # n/a

To test: n/a

CircleCI is [exiting with error code 1](https://app.circleci.com/pipelines/github/wordpress-mobile/WordPress-iOS/12245/workflows/1e19ea83-3a41-49a2-b390-4caa7419020e/jobs/30423/steps) because it wants fastlane to update. In case it's really as straightforward as running `bundle update fastlane`, this PR should fix it.

PR submission checklist:
